packages/iov-bcp-types/src/transactions.ts

```diff
@@ -26,8 +26,11 @@ export interface Amount {
   readonly tokenTicker: TokenTicker;
 }
 
+/** the primary identifier for a registered blockchain in the BNS */
+export type BnsBlockchainId = Uint8Array & As<"blockchain-nft-id">;
```

Comment:
So, you want to separate this from ChainId....
I think we will want them to be the same

Reply:
We use `ChainId` to implement `BnsBlockchainId`. But having the types as separate concepts, the implementation of `BnsBlockchainId` can change if we want